kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java

```diff
@@ -725,6 +734,7 @@ else if (ofType instanceof Set)
 			if (criteria != null) fieldCount += 1;
 			if ((type == SelectionType.JOINABLE) || (type == SelectionType.ACCESS) || (type == SelectionType.UPDATE) || (type == SelectionType.MEMBER) || (type == SelectionType.DELETED) || (type == SelectionType.INACTIVE_ONLY)) fieldCount++;
 			if (propertyCriteria != null) fieldCount += (2 * propertyCriteria.size());
+			if(excludedSites != null && excludedSites.size() > 0) { fieldCount += excludedSites.size(); }
```

Comment:
minor: `!excludedSites.isEmpty()` is more concise when checking an empty collection